tests/measurements/test_probs.py

```diff
@@ -38,6 +66,7 @@ def _init_state(n):
     return _init_state
 
 
+# pylint: disable=attribute-defined-outside-init, redefined-outer-name
```

Comment:
I think you can remove the `attribute-defined-outside-init` disable flag.

Reply:
Done!